internal/mesh/internal/mappers/sidecarproxymapper/mapper.go

```diff
@@ -0,0 +1,67 @@
+package sidecarproxymapper
```

Comment:
One request I would also have is to put each mapper into their own package unless they for some reason need to share some functionality.

Reply:
I'll split it up in a separate PR to reduce conflicts. I think I might also want to combine it with caches.